pilot/platform/kube/controller.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	"istio.io/istio/pilot/model"
+	"istio.io/istio/pilot/proxy"
```

Comment:
This is bad. We typically don’t let any proxy code into this package and vice versa. So if you really need the node construct here, please move the node into the service model in the same PR

Reply:
`proxy` is a generic package. We don't allow kube-envoy coupling, but that package is not envoy specific.